js/RecentDonations.js

```diff
@@ -0,0 +1,196 @@
+(function() {
+var app = angular.module('RecentDonations', []);
+app.controller('RecentDonationsControl', function($scope) {
+
+    var urlVars = GLOBALS.getUrlVars();
+    var count = Number(urlVars['count']);
+    var uid = urlVars['id'];
+    var isTeam = urlVars['team'];
+
+    if (count === undefined) { count = 5 }
+
+    var contributions = []
```

Comment:
:| :| :| You're probably right, I'm used to swift where the rule is that semicolons are only supposed to be used when it's otherwise ambiguous (and it's an error to do otherwise). I can semicolon my JS if you guys want